src/component/absensi/shift/Shift.vue

```diff
@@ -19,74 +17,93 @@
                 </tr>
             </thead>
             <tbody>
-                <tr>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-
+                <tr v-for="item in shiftItems"
+                    :key="item.id"
+                    :item="item">
```

Comment:
These two attributes (`key` and `item`) are unnecessary; only to be used when `v-for`  is [used on a component](https://vuejs.org/v2/guide/list.html#Components-and-v-for)